bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -112,95 +112,95 @@ metadata:
         operators:
         - name: ibm-licensing-operator
           namespace: placeholder
-          channel: beta
+          channel: v3
```

Comment:
@Daniel-Fan We don't need to update channels in this PR, v3 will be a release channel. I believe we still need to use the `beta` channel for the dev purpose.

Reply:
I can update channels and catalogsource in another PR.

Reply:
Could I understand that we only need to update the channel to `v3` in subscription of operator owned by installer?

Reply:
My concern is if we update the channel to v3, it will break the CICD daily build.
My understanding is we just remove the `replace` in the CSV and leave the channel updating works to CICD when they build catalog.

Reply:
@horis233 Please review again, we only remove the `replace` section and keep `olm.skipRange: '>=3.3.0 <3.7.0'`